OHIF/Viewers
# fix(toolbar): allow customizable toolbar for active viewport and allow active tool to be deactivated via a click

### Context

Some tools like crosshairs appear to be more toggle based tools, but are also tool based in that they link a specific button.  This change allows a default tool to be set which will be activated to turn the tool off.  As well, some tools are not applicable
in all tool group ids, so this change allows the menu to be set per tool group.

### Changes & Results

Apply the defaultTool to turn off an active tool.

### Testing

1. Run the viewer in basic-test-mode.
2. Launch MPR.
3. Notice that the toolbar changes to add the crosshairs tool
4. Use the crosshairs tool.
5. Click the crosshairs tool to deactivate it.
6. The window level tool should now be active.



### Checklist

#### PR



- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates



- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
